api/envoy/api/v2/route/route.proto

```diff
@@ -1062,6 +1066,12 @@ message DirectResponseAction {
   core.DataSource body = 2;
 }
 
+message Fallthru {
+  // If this flag is set to 1 then route name (if exists for the route) for fallthru routes will be
```

Comment:
"Set to true"

Per my last comment, stream info is not a useful end user thing to describe. How is the user going to interact with this? Can you describe fully how the access log will work? This will also need to be added to the gRPC access logs in some way?